corehq/ex-submodules/dimagi/utils/dates.py

```diff
@@ -479,3 +479,16 @@ def safe_strftime(val, fmt):
     return safe_val.strftime(fmt
                              .replace("%Y", str(val.year))
                              .replace("%y", str(val.year)[-2:]))
+
+
+def get_start_and_end_dates_of_month(month, year):
```

Comment:
It would be more consistent with `datetime.date` and `datetime.datetime` if the order of these arguments was reversed.